definitions/EiffelMetaProperty/3.1.0.yml

```diff
@@ -0,0 +1,194 @@
+          The identity of the serializer software used to construct the event,
+          in [purl format](https://github.com/package-url/purl-spec).
+        _format: >-
+          [purl specification](https://github.com/package-url/purl-spec)
+        type: string
+        pattern: '^pkg:'
+      uri:
+        _description: The URI of, related to or describing the event sender.
+        _format: URI
+        type: string
+    additionalProperties: false
+  security:
+    _description: >-
+      An optional object for enclosing security related information,
+      particularly supporting data integrity. See
+      [Security](../eiffel-syntax-and-usage/security.md) for further
+      information.
+    type: object
+    properties:
+      authorIdentity:
+        _description: >-
+          The identity of the author of the event. This property is intended
+          to enable the recipient to identify the author of the event contents
+          and/or look up the appropriate public key for decrypting the
+          __meta.security.integrityProtection.signature__ value and thereby
+          verifying author identity and data integrity.
+        _format: '[Distinguished Name](https://tools.ietf.org/html/rfc2253)'
+        type: string
+      integrityProtection:
+        _description: >-
+          An optional object for enabling information integrity protection via
+          cryptographic signing. To generate a correct
+          __meta.security.integrityProtection__ object:
+            1. Generate the entire event, but with the
+               __meta.security.integrityProtection.signature__ value set to
+               an empty string.
+            1. Serialize the event on
+               [Canonical JSON Form](https://tools.ietf.org/html/draft-staykov-hu-json-canonical-form-00).
+            1. Generate the signature using the
+               __meta.security.integrityProtection.alg__ algorithm.
+            1. Set the __meta.security.integrityProtection.signature__ value to
+               the resulting signature while maintaining Canonical JSON Form.
+          To verify the integrity of the event, the consumer then resets
+          __meta.security.integrityProtection.signature__ to an empty string
+          and ensures Canonical JSON Form before verifying the signature.
+        type: object
+        properties:
+          signature:
+            _description: The signature produced by the signing algorithm.
+            type: string
+          alg:
+            _description: >-
+              The cryptographic algorithm used to digitally sign the event.
+              If no signing is performed, the
+              __meta.security.integrityProtection__ SHALL be omitted rather
+              than setting __meta.security.integrityProtection.alg__ to "none".
+            _format: >-
+              [A valid JWA RFC 7518 alg parameter value](https://tools.ietf.org/html/rfc7518#section-3.1),
+              excluding "none"
```

Comment:
You've made some kind of change to this line (visible when I diff this file against 3.0.0.yml) and I suppose it wasn't intentional.

Reply:
Hmm, seems my editor automatically removes trailing whitespaces... Are the two spaces in the end of line important? If so, I can revert them back.

Reply:
Yes they are important. Reference: https://www.markdownguide.org/basic-syntax/#line-breaks